objects/obj_arrowL/Create_0.gml

```diff
@@ -0,0 +1,3 @@
+/// @description Initialize strings by column
+
+height_text = "<----"
```

Comment:
Add a semi-colon at the end of all the height test lines ex.(height_text = "<----";)